tests/test_config.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2016-2017 Facundo Batista, María Andrea Vignau
+#
+# This program is free software: you can redistribute it and/or modify it
+# under the terms of the GNU General Public License version 3, as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranties of
+# MERCHANTABILITY, SATISFACTORY QUALITY, or FITNESS FOR A PARTICULAR
+# PURPOSE.  See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+# For further info, check  https://github.com/facundobatista/recordium
+
+import os
+from unittest import TestCase
+
+from recordium import config
+
+class TestConfig(TestCase):
+    """Tests for config module."""
+
+    def setUp(self):
+        """Set up. Creates a fake config"""
+        self.config_test = _Config()
+
+    def test_set(self):
+        """Basic test: setting and getting a key"""
+        newvalue = 15
+        self.assertNotEqual(self.config_test.POLLING_TIME, newvalue)
+        self.config_test.POLLING_TIME = newvalue
+        self.assertEqual(self.config_test.POLLING_TIME, newvalue)
+
+    def test_attribute_error(self):
+        """Try to access an inexistent attribute to raise an error"""
+        self.assertRaises(AttributeError, exec("self.config_test.SARASA = 1"))
```

Comment:
Because there is no need of calling it. "exec" is a corner case instruction for when you need to execute some code in particular conditions. Why not do something more simple?